### PR TITLE
Update 08-parallel-routes.mdx

### PR DESCRIPTION
**Pull request by hadamalush**
I removed async from 'use client' and added a little changes, because in Nextjs we can't use async with client components.

### CODE REVIEW

File: docs/02-app/01-building-your-application/01-routing/08-parallel-routes.mdx [hunk 1]
```diff
@@ -131,13 +144,22 @@ export default async function Layout(props: {
...
+}
+```
+
 ```jsx filename="app/layout.js" switcher
 'use client'
 
 import { useSelectedLayoutSegment } from 'next/navigation'
 
-export default async function Layout(props) {
+export default function SelectedLayoutSegment(props) {
```
Comment (leerob) [inline docs/02-app/01-building-your-application/01-routing/08-parallel-routes.mdx:160]:
```suggestion
export default function Layout(props) {
```

File: docs/02-app/01-building-your-application/01-routing/08-parallel-routes.mdx [hunk 2]
```diff
@@ -131,13 +144,22 @@ export default async function Layout(props: {
...
 ```
 
+```jsx filename="app/layout.js" switcher
+
+export default async function Layout(props) {
+
+return <SelectedLayoutSegment segment={props.auth}/>
+}
+```
+
```
Comment (leerob) [inline docs/02-app/01-building-your-application/01-routing/08-parallel-routes.mdx:154]:
```suggestion
```

File: docs/02-app/01-building-your-application/01-routing/08-parallel-routes.mdx [hunk 3]
```diff
@@ -117,12 +117,25 @@ On reload, Next.js will first try to render the unmatched slot's `default.js` fi
...
+
 ```tsx filename="app/layout.tsx" switcher
 'use client'
 
 import { useSelectedLayoutSegment } from 'next/navigation'
 
-export default async function Layout(props: {
+export default function SelectedLayoutSegment(props: {
```
Comment (leerob) [inline docs/02-app/01-building-your-application/01-routing/08-parallel-routes.mdx:138]:
```suggestion
export default function Layout(props: {
```

File: docs/02-app/01-building-your-application/01-routing/08-parallel-routes.mdx [hunk 4]
```diff
@@ -117,12 +117,25 @@ On reload, Next.js will first try to render the unmatched slot's `default.js` fi
...
+  //...
+  auth: React.ReactNode
+}) {
+
+return <SelectedLayoutSegment segment={props.auth}/>
+
+}
+```
+
+
```
Comment (leerob) [inline docs/02-app/01-building-your-application/01-routing/08-parallel-routes.mdx:132]:
```suggestion
```

**Review by leerob — APPROVED** [review]

**Comment by leerob** [comment]
Looks like Prettier failed here 🙏

**Comment by samcx** [comment]
@hadamalush closing this since I believe this has since been fixed!

**Pull request closed without merging**